packages/react/src/themes/types.ts

```diff
@@ -209,7 +209,18 @@ export interface ICSSPseudoElementStyle extends ICSSInJSStyle {
   content?: string
 }
 
-export interface ICSSInJSStyle extends React.CSSProperties {
+export interface StardustAnimationName {
+  keyframe?: any
+  params?: object
+}
+
+export type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>
```

Comment:
```suggestion
type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>
```

This type is never used expect this file